src/agent/debuglet.ts

```diff
@@ -175,6 +179,9 @@ export class Debuglet extends EventEmitter {
 
     /** @private {Object.<string, Boolean>} */
     this.completedBreakpointMap_ = {};
+
+    /** @private {Array<number>} */
```

Comment:
Why are we putting this JsDoc everywhere when this is typescript? Maybe because this code has been migrated from JS?

And also why all those trailing underscores in private member names?

Reply:
First question -- yes 
Regarding the names, actually another PR is addressing right now. 
https://github.com/GoogleCloudPlatform/cloud-debug-nodejs/pull/344

I will remove the trailing underscore for this variable here.

Reply:
Please delete this JsDoc (or at least make it correct). It's not a number array. Bonus point if you delete the underscore too :)

Reply:
Done.